spec.html

```diff
@@ -31793,7 +31793,7 @@ <h1>CharacterClassEscape</h1>
         <p>The production <emu-grammar>UnicodePropertyValueExpression :: LoneUnicodePropertyNameOrValue</emu-grammar> evaluates as follows:</p>
         <emu-alg>
           1. Let _s_ be SourceText of |LoneUnicodePropertyNameOrValue|.
-          1. If ! UnicodeMatchPropertyValue(`"General_Category"`, _s_) is identical to a List of Unicode code points that is the name of a Unicode general category or general category alias listed in the &ldquo;Property value and aliases&rdquo; column of <emu-xref href="#table-unicode-general-category-values"></emu-xref>, then
+          1. If ! UnicodeMatchPropertyValue(*"General_Category"*, _s_) is identical to a List of Unicode code points that is the name of a Unicode general category or general category alias listed in the &ldquo;Property value and aliases&rdquo; column of <emu-xref href="#table-unicode-general-category-values"></emu-xref>, then
```

Comment:
Found another! `UnicodeMatchPropertyValue` "takes two parameters ..., each of which is a List of Unicode code points". So here, "General_Category" represents a List of code points, not a String value. (Ditto the occurrence on the next line.) This case should probably be handled in #1724.

Reply:
Good catch! I don't think the next line applies though, as it's referring to [this](https://unicode.org/reports/tr44/#UnicodeData.txt).

Reply:
>  I don't think the next line applies though

Okay, I can go along with that.